Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/isHoisted.md

$isHoisted[roleID]
```

### Parameters

> - `roleID` `(Type : Snowflake || Flag : Required)` : The role which the bot checks is hoisted or not.

Reviewer: ```suggestion
- `roleID` `(Type: Snowflake || Flag: Required)`: The role to check its hoisted status.
```
